assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -275,6 +292,18 @@ class PageEdit extends Component {
 					</PanelColorSettings>
 					<PanelBody title={ __( 'Background Media', 'amp' ) }>
 						<>
+							{
+								isExcessiveVideoSize &&
+								<Notice status="warning" isDismissible={ false } >
+									{
+										sprintf(
+											/* translators: %d: the number of recommended megabytes per second */
+											__( 'A video size of less than %d MB per second is recommended.', 'amp' ),
```

Comment:
Minor thing but it would be useful to know how many MB/second the selected video is, to know how far from the target the video is.